### PR TITLE
Adding PPSSPP logo (original and gold)

### PR DESCRIPTION
**Pull request by indra87g**
And also, I made bold text for your name (Henrik Rydgård)

### CODE REVIEW

**Comment by indra87g** [comment]
Hope you accept it

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/gPybnRP/pngwing-com.png" alt="PPSSPP Original Logo" width="128" height="128"></a>
```
Comment (Florin9doi) [inline README.md:1]:
```suggestion
<img src="source_assets/image/icon_regular.png" alt="PPSSPP Original Logo" width="128" height="128">
<img src="source_assets/image/icon_gold.png" alt="......
```

The images are already present in this repo, there is no need to mirror them

File: README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/8mFD49P/pngwing-com-1.png" alt="PPSSPP Gold Logo - Buy Now !" border="0" width="128" height="128"></a>
+
 PPSSPP - a fast and portable PSP emulator
 =========================================
 
-Created by Henrik Rydgård
+Created by **Henrik Rydgård**
```
Comment (hrydgard) [inline README.md:7]: don't really need my name bolded :)

**Comment by LethalManBoob** [comment]
do not redeem this

**Comment by sneedsfeed1488** [comment]
sars you bully little kid
accept the indra
sar

**Comment by Fijxu** [comment]
what?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/gPybnRP/pngwing-com.png" alt="PPSSPP Original Logo" width="128" height="128"></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/8mFD49P/pngwing-com-1.png" alt="PPSSPP Gold Logo - Buy Now !" border="0" width="128" height="128"></a>
```
Comment (sandboxgamedev123) [inline README.md:2]:
```suggestion
```

Comment (sandboxgamedev123) [reply README.md:2]: I recommend one logo. In this case, the regular logo.

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sandboxgamedev123) [inline README.md:2]:
```suggestion
```
If this pull request is merged, I would recommend one logo. In this case, the regular logo.

**Review by sandboxgamedev123 — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
There is no comment regarding the approval of the pull request.

**Review by ghost — CHANGES_REQUESTED** [review]
I do not know if there is a way to remove previous approvals. There is no comment regarding the approval of the pull request.

**Pull request closed without merging**